src/status_im/notifications_center/core.cljs

```diff
@@ -19,15 +19,16 @@
                 {:unread-count (get db :activity.center/notifications-count 0)
                  :notifications (get-in db [:activity.center/notifications :notifications])}
                 activities)]
-    {:db (-> db
-             (assoc-in [:activity.center/notifications :notifications] notifications)
-             (assoc :activity.center/notifications-count (max 0 unread-count)))
-     :dispatch (cond
-                 (= (:view-id db) :notifications-center)
-                 [:mark-all-activity-center-notifications-as-read]
-
-                 (= (:view-id db) :chat)
-                 [:accept-all-activity-center-notifications-from-chat (:current-chat-id db)])}))
+    (merge
```

Comment:
this part is not related to this issue. it will remove null dispatch error message from the metro. 
is it ok to keep this in this PR or create a separate one?